docs/documentation/special-fields/ttl-cascade.md

```
@@ -130,6 +130,116 @@ await library.aset_ttl(3600, cascade=True)
+
+In both cases the edge carries **all** candidate classes, and the cascade resolves
+each reached child's *actual* class at traversal time from its stored key's
+`{class}:{pk}` prefix, then re-arms that child to **its own** resolved class's
+`Meta.ttl` — exactly as with a single-target FK. A child whose class is a valid model
+but is *not* among the edge's candidates is skipped (never re-armed) and counted as
+class drift in `CascadeResult.mismatched_class`.
+
+The single runnable example below shows **both** shapes — a union FK and a
+polymorphic-base FK — each re-arming its reached child to that child's own `Meta.ttl`:
+
+```python
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)